DEVELOPERS.md

```diff
@@ -35,10 +50,14 @@ Note: just commands will only work inside git-bash shell by default, as they ass
 
 ## Local development environment
 
-Set up a local development environment with:
+Run the Django devserver with:
+```
+just run
+```
+
+Make sure `DJANGO_VITE_DEV_MODE` is set in your environment and run the Vite devserver with:
```

Comment:
It would be worth mentioning that this is an optional step and explaining why you might need to run this. For example, up to this point, I've only made electron changes or tweaked the logging, so this step hasn't been relevant.